echo/server/dembrane/audio_lightrag/utils/audio_utils.py

```
@@ -0,0 +1,106 @@
+import os
+import base64
+import logging
+from io import BytesIO
+
+from pydub import AudioSegment
+
+from dembrane.s3 import save_audio_to_s3, get_stream_from_s3, get_file_size_from_s3_mb
+from dembrane.directus import directus, create_directus_segment, delete_directus_segment
```

Ruff (0.8.2)

`dembrane.directus.delete_directus_segment` imported but unused

Remove unused import: `dembrane.directus.delete_directus_segment`

(F401)